lib/scryptenc/scryptenc.c

```diff
@@ -398,9 +398,12 @@ scryptdec_setup(const uint8_t header[96], uint8_t dk[64],
  * scryptenc_buf(inbuf, inbuflen, outbuf, passwd, passwdlen,
  *     params, verbose, force):
  * Encrypt ${inbuflen} bytes from ${inbuf}, writing the resulting
- * ${inbuflen} + 128 bytes to ${outbuf}.  The explicit parameters
- * within ${params} must be zero or must all be non-zero.  Return
- * the explicit parameters used via ${params}.
+ * ${inbuflen} + 128 bytes to ${outbuf}.  If ${force} is 1, do not check
+ * whether decryption will exceed the estimated available memory or time.
+ * The explicit parameters within ${params} must be zero or must all be
+ * non-zero.  If explicit parameters are used and the computation is estimated
+ * to exceed resource limits, print a warning to stderr instead of returning
```

Comment:
That's not true; `warn0` can now send messages to syslog instead.  Maybe replace
```
print a warning to stderr
```
with
```
issue a warning
```
?

Reply:
Let's drop the "to stderr" but keep "print a warning".

Reply:
And in the description of `scryptenc_file` too.